@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -1672,7 +1677,13 @@ export class TextElementsRenderer {
                 textRenderState!.reset();
             }
 
-            const textRejected = placeResult === PlacementResult.Rejected;
+            const iconIsRequired = !(poiInfo?.iconIsOptional === false);
+            const iconIsReady = poiInfo?.isValid === true;
```

Comment:
could you use the iconReady constant from line 1618? 
https://github.com/heremaps/harp.gl/pull/1888/commits/572d9eb15efd00d3463c2f94e6588ce43c810a63#diff-6b6f266bec84f376c6f75070ad562145d6e14b8e4031e0e34c85be854c939f39R1618